### PR TITLE
Fix Twemoji not loading on Windows

### PR DESCRIPTION
**Pull request by turt2live**

### CODE REVIEW

File: src/utils/FontManager.js [hunk 1]
```diff
@@ -77,7 +77,11 @@ export async function fixupColorFonts() {
     }
 
     if (await isColrFontSupported()) {
-        const path = `url('${require("../../res/fonts/Twemoji_Mozilla/TwemojiMozilla-colr.woff2")}')`;
+        let path = `url('${require("../../res/fonts/Twemoji_Mozilla/TwemojiMozilla-colr.woff2")}')`;
+
+        // Normalize slashes because Windows is made of fail
+        path = path.replace(/\\/g, '/');
```
Comment (bwindels) [inline src/utils/FontManager.js]: should we rather do this in the webpack config script for all requires?

Comment (turt2live) [reply src/utils/FontManager.js]: is that a thing we have? I wasn't sure ._.

**Comment by turt2live** [comment]
> I can make an attempt next week if you don't have time.

Please do - feel free to reuse the branch or start anew

**Comment by jryans** [comment]
https://github.com/vector-im/riot-web/pull/9869 should resolve this issue.

**Pull request closed without merging**